src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
```

Comment:
I think it is better to use other constructor of `Ball` class, which support color and number setting. To simplify it to one line method. (If we do not need to keep `Ball` variable, we can simply:
 `...return new Ball(randomColor, randomNumber);`